library/core/src/main/java/com/google/android/exoplayer2/video/VideoFrameReleaseTimeHelper.java

```diff
@@ -68,19 +74,12 @@ public VideoFrameReleaseTimeHelper() {
    */
   public VideoFrameReleaseTimeHelper(Context context) {
     this(getDefaultDisplayRefreshRate(context));
+    this.context = context.getApplicationContext();
+    registerDisplayListener();
```

Comment:
As far as I can see, there is no need to do this here, or call getDefaultDisplayRefreshRate or setSync anywhere prior to enable being called.

Reply:
If we are going to keep the default constructor (and the DISPLAY_REFRESH_RATE_UNKNOWN functionality) we need to at least call setSync in the constructor since enable only does anything if useDefaultDisplayVsync==true and after which we would reset the refresh rate and then register the listener.